opendrift/models/basemodel.py

```diff
@@ -2786,7 +2783,7 @@ def set_up_map(self, corners=None, buffer=.1, delta_lat=None,
                 logger.debug('Using custom shapes for plotting land..')
                 ax.add_geometries(self.readers['shape'].polys, ccrs.PlateCarree(), facecolor=land_color, edgecolor='black')
             else:
-                opendrift.readers.reader_global_landmask.plot_land(ax, lonmin, latmin, lonmax, latmax, fast, ocean_color, land_color, lscale)
+                reader_global_landmask.plot_land(ax, lonmin, latmin, lonmax, latmax, fast, ocean_color, land_color, lscale)
```

Comment:
For unknown (to me) reason, the above change had to be made.
Then instead of having separate imports of reader_global_landmask throughout Basemodel, I made one at the top. But a but shaky if this change should lead to apparently unrelated failures?

Reply:
Ok, either way should work.